pom.xml

```diff
@@ -165,7 +165,7 @@
         <dependency>
             <groupId>org.smartregister.hapi-fhir-opensrp-extensions</groupId>
             <artifactId>practitioner</artifactId>
-            <version>0.0.11-SNAPSHOT</version>
+            <version>0.0.12-SNAPSHOT</version>
```

Comment:
Shouldn't this read `v0.0.20-SNAPSHOT`?

Reply:
No @ndegwamartin . We have a separate `pom.xml` for each module. We only upgraded the practitioner module hence` 0.0.12 version.`